lte/gateway/docker/services/trfgen/Dockerfile

```diff
@@ -0,0 +1,48 @@
+################################################################################
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+ARG OS_DIST=ubuntu
+ARG OS_RELEASE=focal
+
+FROM $OS_DIST:$OS_RELEASE
+
+ARG MAGMA_RELEASE=master
+ARG DEBIAN_FRONTEND=noninteractive
+
+ARG ETHTOOL_VERSION=1:5.4-1
+ARG GIT_VERSION=1:2.25.1-1ubuntu3.4
+ARG IPERF3_VERSION=3.7-3
+ARG PYTHON3_PIP_VERSION=20.0.2-5ubuntu1.6
+ARG PYTHON3_VENV_VERSION=3.8.2-0ubuntu2
+ARG VIRTUALENV_VERSION=20.0.17-1ubuntu0.4
+
+ENV MAGMA_ROOT=/magma
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  ethtool=$ETHTOOL_VERSION \
+  git=$GIT_VERSION \
+  iperf3=$IPERF3_VERSION \
+  python3-pip=$PYTHON3_PIP_VERSION \
+  python3-venv=$PYTHON3_VENV_VERSION \
+  virtualenv=$VIRTUALENV_VERSION \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/* && \
+  pip3 install --no-cache-dir git+https://github.com/illu89/iperf3-python#egg=iperf3-python \
+  pyroute2 \
+  scapy-python3 && \
+  git clone https://github.com/magma/magma $MAGMA_ROOT -b $MAGMA_RELEASE && \
+  mkdir -p /usr/lib/python3.8/util && \
+  cp $MAGMA_ROOT/lte/gateway/deploy/roles/trfserver/files/traffic_server.py /usr/local/bin/traffic_server.py && \
+  cp $MAGMA_ROOT/lte/gateway/deploy/roles/trfserver/files/util/traffic_messages.py /usr/lib/python3.8/util/
+
+ENTRYPOINT /usr/local/bin/traffic_server.py
```

Comment:
Currently if I run the the container it seems to run against `127.0.0.1` instead of 192.168.60.144:
```
$ docker run -it agw_trfgen:latest
20220624-191849.221 [PID 7] DEBUG (127.0.0.1:62462) Initializing dispatcher...
20220624-191849.221 [PID 7] DEBUG (127.0.0.1:62462) Setting log level to: DEBUG
20220624-191849.221 [PID 7] INFO (127.0.0.1:62462) Dispatcher running on 127.0.0.1:62462 (non-daemon mode)
```
with 
```
$ docker inspect agw_trfgen:latest
[...]
            "Cmd": [
                "192.168.60.144",
                "62462"
            ],
[...]
            "Entrypoint": [
                "/bin/sh",
                "-c",
                "/usr/local/bin/traffic_server.py"
            ],
[...]
```


If I apply the suggested change here:

```suggestion
ENTRYPOINT ["/usr/local/bin/traffic_server.py"]
```
it seems to work:
```
$ docker run --net host -it agw_trfgen:latest 192.168.1.121 62462
20220624-192533.856 [PID 1] DEBUG (192.168.1.121:62462) Initializing dispatcher...
20220624-192533.856 [PID 1] DEBUG (192.168.1.121:62462) Setting log level to: DEBUG
20220624-192533.856 [PID 1] INFO (192.168.1.121:62462) Dispatcher running on 192.168.1.121:62462 (non-daemon mode)
```